src/ResourceManager/AzureBatch/Commands.Batch/Commands.Batch.csproj

```diff
@@ -42,8 +42,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Batch, Version=8.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Azure.Batch.8.1.1\lib\net452\Microsoft.Azure.Batch.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Batch, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Batch.9.0.0\lib\net452\Microsoft.Azure.Batch.dll</HintPath>
```

Comment:
@matthchr you'll want to make sure the version of your SDK in [`Commands.Batch.Netcore.csproj`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/AzureBatch/Commands.Batch/Commands.Batch.Netcore.csproj#L35)

Reply:
Done -- do I need to do any extra testing around this...?

Reply:
@matthchr we're in the process of converting all of the desktop test projects to .NET Standard, so we're looking to run all tests on Travis soon. As long as the coverage is in the desktop version, we should feel fairly confident it works on .NET Standard as well.